external/anomaly/anomaly_classification/task.py

```diff
@@ -227,7 +235,7 @@ def export(self, export_type: ExportType, output_model: ModelEntity):
         with open(xml_file, "rb") as file:
             output_model.set_data("openvino.xml", file.read())
         output_model.set_data("label_schema.json", label_schema_to_bytes(self.task_environment.label_schema))
-        output_model.set_data("threshold", bytes(struct.pack("f", self.model.threshold.item())))
+        output_model.set_data("threshold", bytes(struct.pack("f", self.model.image_threshold.value.item())))
```

Comment:
Similar to above. Is it possible to do the following?

```
output_model.set_data("threshold", self.model.image_threshold.value.cpu().numpy().tobytes())

```

Reply:
can it be "confidence_threshold" as in case of detection. Just would like to have these things aligned

Reply:
hmm, this would be an idea to distinguish the term threshold from image_threshold and pixel_threshold. @djdameln , thoughts?

Reply:
This threshold value is applied to the (standardized) anomaly scores, so technically it is not a confidence threshold. The anomaly scores are converted to confidence values after the threshold is applied. That's why I prefer to keep using `threshold`. I could change it to `image_threshold` to make the distinction between image-level and pixel-level threshold and better match the naming in anomalib.